### DISCUSSION TITLE
✅ 🔗 Possibility to design product cards (Duplicate)

### DISCUSSION THREAD

**Discussion by paradisiacor** [elementor/elementor#16999] [Feature Request]
Hey! Great job with everything you have done so far! 

Is there a slight chance that we might see a product card design feature in the future? It would be an amazing feature.

Cheers!

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the new Loop Builder in Elementor Pro v3.9! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
